packages/appstore/src/components/wallets-consent-popup/wallets-consent-popup.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
```

Comment:
suggest this name for file:` ready-to-update-wallets.tsx`

Reply:
Done 😃 👍🏼